src/Microsoft.OData.Edm/Vocabularies/Values/EdmTimeOfDayConstant.cs

```diff
@@ -4,20 +4,22 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.OData.Edm.Vocabularies
 {
     /// <summary>
-    /// Represents an EDM TimeOfDay constant.
+    /// Represents an EDM TimeOnly constant.
     /// </summary>
     public class EdmTimeOfDayConstant : EdmValue, IEdmTimeOfDayConstantExpression

```

Comment:
Would there be any value in naming this `EdmTimeOfDayConstant` to `EdmTimeOnlyConstant` or would it just result into unnecessary code churn? Note that the class is public. However, this is the one case that I'd probably be inclined towards renaming.